servers/user-api/src/dataService/userDataService.ts

```diff
@@ -142,17 +142,20 @@ export class UserDataService {
         'users.user_id',
       )
       .leftJoin(
-        this.readDb.raw(`(
-        SELECT user_id,
```

Comment:
this was just grabbing the first row returned by the ordering (expires_at desc, active desc) regardless of user_id, and limiting to 1

so the subsequent join would return null (and set the user to be 'NEVER')